frame/nomination-pools/src/lib.rs

```diff
@@ -2257,8 +2257,8 @@ impl<T: Config> Pallet<T> {
 		current_points: BalanceOf<T>,
 		new_funds: BalanceOf<T>,
 	) -> BalanceOf<T> {
-		let u256 = |x| T::BalanceToU256::convert(x);
```

Comment:
not looking more readable to my eyes

Reply:
`u256` and `balance` are odd function names. Have renamed to `to_u256` and `to_balance` for readability.